### PR TITLE
fix: support m.youtube.com subdomain links

### PR DESCRIPTION
**Pull request by verbiricha**
https://snort.social/e/note1l8xjvck3cqa9ghlccf2vnsmdnp0vak7uam9h7uyqpzuggdrnj80qs0pfxq

### CODE REVIEW

**Comment by v0l** [comment]
How do we make this resize properly

**Comment by v0l** [comment]
> How do we make this resize properly https://youtube-m.snort-social.pages.dev/e/note1l8xjvck3cqa9ghlccf2vnsmdnp0vak7uam9h7uyqpzuggdrnj80qs0pfxq

```css
    display: block;
    margin: 10px auto;
    width: -webkit-fill-available;
    aspect-ratio: 16 / 9;
```

**Comment by v0l** [comment]
> > How do we make this resize properly https://youtube-m.snort-social.pages.dev/e/note1l8xjvck3cqa9ghlccf2vnsmdnp0vak7uam9h7uyqpzuggdrnj80qs0pfxq
> 
> ```css
>     display: block;
>     margin: 10px auto;
>     width: -webkit-fill-available;
>     aspect-ratio: 16 / 9;
> ```

image

File: src/element/Note.css [hunk 1]
```diff
@@ -32,12 +32,15 @@
...
     max-height: 500px;
-    margin: 10px;
-    margin-left: auto;
-    margin-right: auto;
+    margin: 10px auto;
     display: block;
 }
 
+.note > .body iframe, note > .body video {
+    width: -webkit-fill-available;
```
[the remark this answers is not in the document]
Comment (verbiricha) [reply src/element/Note.css:40]: Added in [236da05](https://github.com/v0l/snort/pull/27/commits/236da0561770a9ad9679f9ec2657e6c73b247221)

**Pull request merged**